react/features/welcome/actions.js

```diff
@@ -1,6 +1,26 @@
 // @flow
 
-import { SET_SIDEBAR_VISIBLE } from './actionTypes';
+import {
+    SET_SIDEBAR_VISIBLE,
+    SET_WELCOME_PAGE_LIST_DEFAULT_PAGE
+} from './actionTypes';
+
+/**
+ * Action to update the default page index of the {@code WelcomePageLists}
+ * component.
+ *
+ * @param {number} pageIndex - The index of the selected page.
+ * @returns {{
+ *     type: SET_WELCOME_PAGE_LIST_DEFAULT_PAGE,
+ *     pageIndex: number
+ * }}
+ */
+export function setWelcomePageListDefaultPage(pageIndex: number) {
+    return {
+        type: SET_WELCOME_PAGE_LIST_DEFAULT_PAGE,
```

Comment:
It's an unfortunate naming with the `s` but what can you do? The component is called `WelcomePageLists` so, according to our coding style, it should be referenced everywhere by the same name: `SET_WELCOME_PAGE_LISTS_DEFAULT_PAGE`, `setWelcomePageListsDefaultPage`. I saw your name without the `s` and went looking for `WelcomePageList` but then saw `WelcomePageLists` and wasn't sure whether I was to find one without an `s` or you merely left the `s` out for some purpose unknown to me. Consistency removes the unnecessary misunderstanding.